mlflow/server/js/src/components/ExperimentViewUtil.js

```diff
@@ -120,30 +120,36 @@ export default class ExperimentViewUtil {
   /**
    * Returns header-row table cells for columns containing run metadata.
    */
-  static getRunMetadataHeaderCells(onSortBy, sortState, cellType) {
+  static getRunMetadataHeaderCells(onSortBy, curOrderByKey, curOrderByAsc, cellType) {
     const CellComponent = `${cellType}`;
-    const getHeaderCell = (key, text, sortable = true) => {
-      const sortIcon = ExperimentViewUtil.getSortIcon(sortState, false, false, key);
+    const getHeaderCell = (key, text, canonicalSortKey) => {
+      const sortIcon = ExperimentViewUtil.getSortIcon(curOrderByKey, curOrderByAsc,
+        canonicalSortKey);
       return (
         <CellComponent
           key={"meta-" + key}
           className="bottom-row sortable run-table-container"
-          onClick={() => onSortBy(false, false, key)}
+          onClick={() => onSortBy(canonicalSortKey, !curOrderByAsc)}
         >
           <span style={ExperimentViewUtil.styles.headerCellText}>{text}</span>
-          {sortable && <span style={ExperimentViewUtil.styles.sortIconContainer}>{sortIcon}</span>}
+          {canonicalSortKey !== null && 
+            <span style={ExperimentViewUtil.styles.sortIconContainer}>{sortIcon}</span>}
         </CellComponent>);
     };
     return [
-      getHeaderCell("start_time", <span>{"Date"}</span>),
-      getHeaderCell("user_id", <span>{"User"}</span>),
-      getHeaderCell("run_name", <span>{"Run Name"}</span>),
-      getHeaderCell("source", <span>{"Source"}</span>),
-      getHeaderCell("source_version", <span>{"Version"}</span>),
-      getHeaderCell("tags", <span>Tags</span>, false),
+      getHeaderCell("start_time", <span>{"Date"}</span>, "attributes.start_time"),
+      getHeaderCell("user_id", <span>{"User"}</span>, "tags.`mlflow.user`"),
+      getHeaderCell("run_name", <span>{"Run Name"}</span>, "tags.`mlflow.runName`"),
+      getHeaderCell("source", <span>{"Source"}</span>, "tags.`mlflow.source.name`"),
+      getHeaderCell("source_version", <span>{"Version"}</span>, "tags.`mlflow.source.git.commit`"),
+      getHeaderCell("tags", <span>Tags</span>, null),
```

Comment:
For some reason, now clicking on the "Tags" header spuriously sets the sort state in the URL & triggers a search API request - we should probably stick with the behavior in master of not changing the sort state